sky/specs/style-guide.md

```diff
@@ -8,44 +7,114 @@ that everyone, whether reading the code for the first time or
 maintaining it for years, can quickly determine what the code does. A
 secondary goal is avoiding arguments when there are disagreements.
 
-All languages
--------------
+## All languages
 
-Avoid checking in commented-out code. It will bitrot too fast to be
-useful, and will confuse people maintaining the code.
+### AVOID checking in commented-out code
```

Comment:
The answer to everything is: be consistent with the standard style guide. This style guide is a delta on top of the standard Dart guide, which we recommend that people read. Therefore, it makes a lot of sense to preserve the same style. Some additional points:

## Why rules in the headings

- they make it easy to quickly scan the guide and familiarizing yourself with the main points
- they make it easy to copy and paste guidelines into a code review comment

## What's with the choice of headlines

Like in the standard guide, each headline attempts to make the most important succinctly. The paragraphs then expand on the point, give examples, and mention asides. We can certainly switch things up as necessary. I by no means insist that my choice of headlines is the most optimal. I'm simply bootstrapping an idea.